sql/restoring.sql

```diff
@@ -26,7 +26,8 @@ DECLARE
 BEGIN
     SELECT current_database() INTO db;
     EXECUTE format($$ALTER DATABASE %I RESET timescaledb.restoring $$, db);
-    RESET timescaledb.restoring;
+    -- we cannot use reset here because the reset_val might not be off
+    SET timescaledb.restoring TO off;
     PERFORM _timescaledb_internal.restart_background_workers();
```

Comment:
I remember there were [some concerns](https://github.com/timescale/timescaledb/pull/4290#discussion_r863539422) that this will create problems with `pg_dumpall`. Are they still valid with this solution?

Reply:
post_restore would be run after `pg_dumpall` so it cannot have an effect on it